config/docker/security/rotate-secrets.sh

```
@@ -0,0 +1,137 @@
+            return 1
+        fi
+    fi
+}
+
+# Function to rotate JWT secret
+rotate_jwt_secret() {
+    local new_secret=$(generate_random_string 64)
+    save_secret "$SECRETS_DIR/jwt_secret.txt" "$new_secret"
+    
+    # Update Vault if available
+    if [ -n "$VAULT_TOKEN" ] && [ -n "$VAULT_ADDR" ]; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)